include/sway/tree/workspace.h

```diff
@@ -134,4 +134,6 @@ size_t workspace_num_tiling_views(struct sway_workspace *ws);
 
 size_t workspace_num_sticky_containers(struct sway_workspace *ws);
 
+void workspace_squash(struct sway_workspace *workspace);
```

Comment:
Let's add a description of both what this and `container_flatten` does to the headers.